apps/site/src/views/Resources/components/ResourceCard/ResourceCard.module.scss

```diff
@@ -64,4 +69,49 @@
 .text {
 	width: 90%;
 	text-align: center;
+	font-family: utils.$font-family-base;
+}
+
+.islandBackground {
+	position: absolute;
+}
+
+.CardContainer {
+	position: relative;
+	height: auto;
+	aspect-ratio: 1 / 1.3;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	font-family: utils.$font-family-base;
+}
+
+@include utils.media-breakpoint-up(sm) {
+	.CardContainer {
+		width: 500px;
+		min-width: 500px;
+	}
+	.islandBackground {
+		width: 500px;
+	}
+	.group {
+		width: 300px;
+	}
+}
+
+@include utils.media-breakpoint-down(sm) {
+	.CardContainer {
+		width: 350px;
+		min-width: 350px;
+	}
+	.islandBackground {
+		width: 350px;
+	}
+	.group {
+		width: 200px;
+	}
+	.description {
+		max-height: 100px;
+		overflow: auto;
+	}
```

Comment:
Suggestion: to be consistent with Bootstrap's "mobile first" design approach, these css styles can be the default and then you can keep using `media-breakpoint-up` for larger screen sizes.